runtime/tr.source/trj9/env/CpuUtilization.hpp

```diff
@@ -189,7 +189,15 @@ struct TR_CpuEntitlement
    // sure te portlib is up and running
    void init(J9JITConfig *jitConfig)
       {
-      _hypervisorPresent = TR_maybe;
+      /* Couple of issues were discovered when the support for hypervisor was enabled.
+       * When JVM ran on VMWare, then the port library loaded libvmGuestLib.so which
+       * interferes with implementation of some of j.l.Math methods.
+       * The library also causes JVM to core dump when an application is using libjsig.so
+       * for signal chaining.
+       * For these reasons, support for hypervisor is being disabled until the issues with
+       * VMWare library are resolved.
+       */
```

Comment:
Can this comment link to an issue tracking this problem?

Reply:
We don't have an issue as the problems were reported in IBM internal builds which make use of the VMWare library mentioned in the comment. The problem, as such, is not present with OpenJ9 build.